src/styles/components/_component-grid.scss

```diff
@@ -69,6 +179,10 @@ $breakpoint-tablet: 820px;
       &--max-scott {
         grid-area: max-scott;
       }
+
+      border: 1px solid;
```

Comment:
Oh I found those stray borders!! Right now, we get like, a secondary white border on all the component cards, so I would just delete this.

Reply:
Nice catch, just got rid of it! :)